### ISSUE TITLE
zowe config list shows empty secure array

### ISSUE THREAD

**Issue by dkelosky** [zowe/imperative#511]
If I have a `zowe.config.json` and display it with `zowe config list`, I see:
```
secure:
  (empty array)
```

Even thought this is in my config:
```
  "secure": [
        "profiles.my_base.properties.user",
        "profiles.my_base.properties.password"
    ]
}
```

**Comment by t1m0thyj** [comment]
At first glance this seems like it would be easy to fix but I'm not sure it is.

It appears that @tucker01 intentionally left the secure array blank when merging layers:
https://github.com/zowe/imperative/blob/b21781a9c820dfe9f735dcda3f57d4a2c0891f4e/packages/config/src/Config.ts#L452-L455

**Comment by dkelosky** [comment]
I'm not sure if it should be displayed or not.  As an observation, I found it confusing and thought I had something wrong.  

So, we could show the content, or otherwise describe that the output is suppressed.  A favor showing the content but am mostly indifferent.

**Comment by t1m0thyj** [comment]
This is fixed in #611 which moves the secure array:
```yaml
profiles:
  my_base:
    type:       base
    properties:
      rejectUnauthorized: true
      user:               (secure value)
      password:           (secure value)
    secure:
      - user
      - password
```

The way that "user" and "password" appear twice seems a bit redundant, but hopefully it makes it clear how the "secure" array works.

**Comment by t1m0thyj** [comment]
The Imperative fix should show up in Zowe CLI after https://github.com/zowe/zowe-cli/pull/1020 is merged.
